### ISSUE TITLE
Add functionality to support the database addon

### ISSUE THREAD

**Issue by itsderek23** [scoutapp/scout_apm_python#39]
The Python agent instruments database queries, adds tracebacks to slow queries, and identifies N+1s but does not have the dedicated database query view that's available in the Ruby agent.

Add the required functionality to the agent to support the database addon functionality within Scout.

**Comment by pedropregueiro** [comment]
hey @itsderek23, I see you guys already have support for pymongo instrumentation (awesome!), but I'm guessing until this view is done, we won't be able to see much from it. am I correct?

**Comment by pedropregueiro** [comment]
or could it be that I just don't see any DB instrumentation because we're on the free tier (through Heroku)?

**Comment by itsderek23** [comment]
Hi @pedropregueiro - without the db addon, you still see:

* Mongo on timeseries charts
* The sum of time spent across all Mongo queries (and the number of queries) per transaction
* Each Mongo query in a transaction trace

What you don't see:

* A breakdown of your most expensive Mongo queries across all transactions
* Slow Mongo query insights
* Time spent in each type of Mongo query per transaction

In summary, the visibility is still pretty solid but may require a bit more digging around that the addon does for you.

**Comment by pedropregueiro** [comment]
cool, thanks for the info @itsderek23!

**Comment by cscheng** [comment]
Would very much welcome this feature!

**Comment by dlanderson** [comment]
The DB addon has been supported in Python since 1.3.8 (in September 2018).
